io.elementary.feedback.yml

```
@@ -0,0 +1,79 @@
+app-id: io.elementary.feedback
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+base: io.elementary.BaseApp
+base-version: juno-20.08
+sdk: org.gnome.Sdk
+command: io.elementary.feedback
+finish-args:
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+
+  # needed for perfers-color-scheme
+  - '--system-talk-name=org.freedesktop.Accounts'
+
+  - '--metadata=X-DConf=migrate-path=/io/elementary/feedback/'
+modules:
+  - name: appstream
+    buildsystem: meson
+    config-opts:
+      - '-Dvapi=true'
+    sources:
+      - type: git
+        url: https://github.com/ximion/appstream
+        tag: 'v0.12.11'
+    modules:
+      - name: xapian
+        sources:
+          - type: archive
+            url: https://oligarchy.co.uk/xapian/1.4.17/xapian-core-1.4.17.tar.xz
+            md5: 64cd582a4495c5cbda8e06e63f8ce4f4
+      - name: yaml
+        sources:
+          - type: git
+            url: https://github.com/yaml/libyaml
+            tag: '0.2.5'
+      - name: Intltool
+        sources:
+          - type: archive
+            url: https://launchpad.net/intltool/trunk/0.51.0/+download/intltool-0.51.0.tar.gz
+            md5: 12e517cac2b57a0121cda351570f1e63
+      - name: xmlto
+        buildsystem: autotools
+        sources:
+          - type: archive
+            url: https://releases.pagure.org/xmlto/xmlto-0.0.28.tar.bz2
+            md5: 93bab48d446c826399d130d959fe676f
+      - name: lmdb
+        buildsystem: simple
+        build-commands:
+          - 'make'
+          - 'make install prefix=/app'
```

```suggestion
        no-autogen: true
        make-install-args:
          - 'prefix=/app'
```

No need to use simple here.